processor/signozspanmetricsprocessor/processor.go

```diff
@@ -475,9 +479,10 @@ func (p *processorImp) exportMetrics(ctx context.Context) {
 		p.logger.Error("Failed to build metrics", zap.Error(err))
 	}
 
-	if err := p.metricsConsumer.ConsumeMetrics(ctx, m); err != nil {
-		p.logger.Error("Failed ConsumeMetrics", zap.Error(err))
-		return
+	for _, consumer := range p.metricsConsumer {
+		if err := consumer.ConsumeMetrics(ctx, m); err != nil {
+			p.logger.Error("Failed ConsumeMetrics for exporter", zap.String("exporter", p.config.MetricsExporter), zap.Error(err))
```

Comment:
This is not going to be a helpful log, as it logs the original comma-separated string instead of the specific consumer it failed to send.